### PR TITLE
Bulk Post support for phase 4 migrations

### PR DESCRIPTION
**Pull request by NileshPant1999**

### CODE REVIEW

File: fyle/platform/apis/v1beta/admin/projects.py [hunk 1]
```diff
@@ -2,12 +2,13 @@
 V1 Beta Admin Projects
 """
 
+#from ....internals.post_bulk_resources import PostBulkResources
```
Comment (Sravanksk) [inline fyle/platform/apis/v1beta/admin/projects.py:5]: why is this commented

File: fyle/platform/apis/v1beta/admin/settlements.py [hunk 2]
```diff
@@ -6,9 +6,9 @@
 from ....internals.list_all_resources import ListAllResources
 from ....internals.list_resources import ListResources
 from ....internals.post_resources import PostResources
+from ....internals.post_bulk_resources import PostBulkResources
```
Comment (Sravanksk) [inline fyle/platform/apis/v1beta/admin/settlements.py:9]: we don't have post_bulk for settlements

File: fyle/platform/apis/v1beta/admin/reimbursements.py [hunk 3]
```diff
@@ -6,9 +6,11 @@
 from ....internals.list_all_resources import ListAllResources
 from ....internals.list_resources import ListResources
 from ....internals.post_resources import PostResources
+from ....internals.post_bulk_resources import PostBulkResources
 
 
-class Reimbursements(ListResources, ListAllResources, PostResources, GetResources):
+
+class Reimbursements(ListResources, ListAllResources, PostResources, GetResources, PostBulkResources):
```
Comment (Sravanksk) [inline fyle/platform/apis/v1beta/admin/reimbursements.py:13]: for reimbursements to do POST BULK, the endpoint changes 
`/admin/reimbursements/mark_paid/bulk`

Can we add a separate method for reimbursements

Comment (NileshPant1999) [reply fyle/platform/apis/v1beta/admin/reimbursements.py:13]: okay will fix this

File: fyle/platform/apis/v1beta/admin/reimbursements.py [hunk 4]
```diff
@@ -8,10 +8,18 @@
 from ....internals.post_resources import PostResources
 
 
+
```
Comment (Sravanksk) [inline fyle/platform/apis/v1beta/admin/reimbursements.py:11]: remove extra line

File: fyle/platform/apis/v1beta/admin/reimbursements.py [hunk 5]
```diff
@@ -8,10 +8,18 @@
...
 class Reimbursements(ListResources, ListAllResources, PostResources, GetResources):
     """Class for Reimbursements APIs."""
 
     REIMBURSEMENTS = '/reimbursements'
+    POST_REIMBURSEMENTS = '/reimbursements/mark_paid/bulk'
```
Comment (Sravanksk) [inline fyle/platform/apis/v1beta/admin/reimbursements.py:16]: `POST_REIMBURSEMENTS_BULK`

File: fyle/platform/apis/v1beta/admin/expense_fields.py [hunk 6]
```diff
@@ -1,13 +1,13 @@
...
 """
-
+from ....internals.post_bulk_resources import PostBulkResources
 from ....internals.list_all_resources import ListAllResources
 from ....internals.list_resources import ListResources
 from ....internals.post_resources import PostResources
 
 
-class ExpenseFields(ListResources, ListAllResources, PostResources):
+class ExpenseFields(ListResources, ListAllResources, PostResources, PostBulkResources):
```
Comment (Sravanksk) [inline fyle/platform/apis/v1beta/admin/expense_fields.py:10]: post bulk is not supported for expense_fields, please remove

**Review by Sravanksk — APPROVED** [review]

**Pull request merged**